packages/flutter/lib/src/rendering/platform_view.dart

```diff
@@ -94,13 +94,13 @@ class RenderAndroidView extends RenderBox {
 
   /// Which gestures should be forwarded to the Android view.
   ///
-  /// The gesture recognizers on this list participate in the gesture arena for each pointer
-  /// that was put down on the render box. If any of the recognizers on this list wins the
+  /// Gesture recognizers created by factories in this set participate in the gesture arena for each
+  /// pointer that was put down on the render box. If any of the recognizers on this list wins the
   /// gesture arena, the entire pointer event sequence starting from the pointer down event
   /// will be dispatched to the Android view.
-  set gestureRecognizers(List<OneSequenceGestureRecognizer> recognizers) {
+  set gestureRecognizers(Set<Factory<OneSequenceGestureRecognizer>> recognizers) {
```

Comment:
the setter should be next to (immediately after) its field. Also, there should be a getter (never have a setter with no getter).